### DISCUSSION TITLE
Show Notification when Offline network in nextjs

### DISCUSSION THREAD

**Discussion by tomdev-FE** [vercel/next.js#46748] [Ideas]
### Goals

I want to show toast message if I turn off network in nextjs, how can i do that?

### Non-Goals

1.
2.
3.

### Background

I want to show toast message if I turn off network in nextjs, how can i do that?

### Proposal

I want to show toast message if I turn off network in nextjs, how can i do that?

**Comment by ciruz** [comment]
It's pretty simple with `window.navigator.onLine`.

Checkout:

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/onLine

There is also a list with compatible browsers, every major browser support this.

Combine this with an event listener and a toast library and you are all set.
